OWML.ModHelper/ModHelper.cs

```diff
@@ -16,9 +16,10 @@ public class ModHelper : IModHelper
         public IModManifest Manifest { get; }
         public IModConfig Config { get; }
         public IOwmlConfig OwmlConfig { get; }
+        public IModInteraction ModInteraction { get; }
```

Comment:
Maybe call it just Interaction here (and in the interface), so it's used like ModHelper.Interaction? Your call :)